themes/wim/components/wim/_block--menu-top-tasks.scss

```diff
@@ -71,7 +76,7 @@
           }
           &:after {
             position: absolute;
-            font-family: 'Glyphicons Halflings';
+            font-family: 'Glyphicons Halflings'; //@todo use fontawesome icons
```

Comment:
@Crazylord Like discussed last time. We decided to use the fontawesome pack, so make use of the right extends.